README.md

```diff
@@ -2,15 +2,379 @@
+
+```bash
+├── Gemfile
+├── README.md
+├── app
+│   ├── controllers
+│   │   └── application_controller.rb
+│   ├── models
+│   │   └── model.rb
+│   └── views
+│       └── index.erb
+├── config
+│   └── environment.rb
+├── config.ru
+├── public
+│   └── stylesheets
+└── spec
+    ├── controllers
+    ├── features
+    ├── models
+    └── spec_helper.rb
+```
+
+### `Gemfile`
+
+This holds a list of all the gems needed to run the application. The bundler gem provides us access to a terminal command: `bundle install`. Bundler will look in the Gemfile and install any gems, as well as any gem dependencies for this application.
+
+Go ahead and enter this command in terminal. It will create a `Gemfile.lock` file for you, which is just a documentation of what versions of the gem you have installed and should use. The lock word is actually because it makes sure that only one thing is running bundle install at a time.
+
+### `app` directory
+
+This folder holds our MVC directories - `models`, `views`, and `controllers`. We spend most of our time coding in this directory.
+
+#### `models` directory
+
+This directory holds the logic behind our application. Typically, these files represent either a component of your application, such as a User, Post, or Comment, or a unit of work. Each file in models typically contains a different class. For example, `dog.rb` would contain a class called `Dog`. As you might have guessed, models represent the "M" components of the MVC paradigm.
+
+Models represent the data and object logic of our application.
+
+Create a new file in the models directory to create a dog class. This class should have name, breed, and age attributes which can be set on initialization. You should be able to read and write to these attributes. This class should also keep track of each instance of dog created, as well as a class method `all` to return an array of those instances.
+
+#### `controllers` directory
+
+The controllers, such as `application_controller.rb`, are where the application configurations, routes, and controller actions are implemented. There is typically a class, which in this case we will call `ApplicationController`, that represents an instance of your application when the server is up and running. The `application_controller.rb` file represents the "C" components of the MVC paradigm.
+
+(In some simple applications -including several labs and code-alongs in this track - the Application Controller will simply be called `app.rb` and will live in the root directory of the project)
+
+Sometimes our other controllers will use `ApplicationController` as an inheritance point so that they inherit all the defaults and behaviors defined in our main `ApplicationController`. Other times our other controllers will simply inherit from `Sinatra::Base`.
+
+Controllers represent the application logic, generally, the interface and flow of our application.
```

Comment:
We said above that models represent the application logic -- could we make the distinction between models and controllers clearer?